### PR TITLE
silex 2 readme

### PR DESCRIPTION
**Pull request by h-bragg**
Update silex 2 readme to indicate the different versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,19 +13,28 @@ appending a slash and issuing an internal sub-request.
 See [silexphp/Silex #149](https://github.com/silexphp/Silex/issues/149) for more
 information about the default Silex routing behavior.
 
+We try to support all commonly used versions of Silex including:
+
+- [Silex 2][silex-2] on [`master`][branch-master] branch, `^2.0` releases
+- [Silex 1.3][silex-1] on [`1.x`][branch-1.x] branch, `^1.0` releases
```
Comment (hkdobrev) [inline README.md]: I'll recommend to change that to copy-pasteable `composer` commands for Silex 1 and Silex 2:

For Silex 2.x:
```shell
composer require graze/silex-trailing-slash-handler
```

For Silex 1.x:
```shell
composer require graze/silex-trailing-slash-handler:^1
```

**Review by hkdobrev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hkdobrev) [reply README.md]: At the moment the usage in code is the same for both Silex 1 and Silex 2 so people won't need to read different documentation for different branches. We can have an Install section with these and Usage would be just about the code usage.

Comment (h-bragg) [reply README.md]: right there with you... :)

**Review by hkdobrev — APPROVED** [review]
👏

**Pull request merged**